packages/data-api/src/__tests__/TupaiaDataApi.test.js

```diff
@@ -13,6 +13,7 @@ describe('TupaiaDataApi', () => {
   const models = getTestModels();
 
   beforeAll(async () => {
+    jest.setTimeout(30000); // Extend default jest timeout as these tests can take a while
```

Comment:
I did run into the same problem when running the indicators test locally and had to do this as well (maybe because my machine is slow). Maybe we should move this to somewhere more global?

Reply:
Yeah I've been having a few issues with performance in jest and @kael89 mentioned he was having it too... Usually the codeship tests run fine despite it however I actually saw this suite timeout in codeship as well one time...

I suspect we haven't got jest configured quite properly... I'd prefer we got to the bottom of the mystery a bit more before globally extending the timeout?

Reply:
Jest is not slow anymore for me (was a local issue). However, often it does get stuck in local integration tests.

I dis some investigating a while ago. My initial conclusions were that it is related to database connection attempts and jest parallelization (we didn't have those issues with mocha, which runs tests serially).

Hope to get some time to investigate and fix this soon!